lib/parsers/text.js

```diff
@@ -52,10 +45,16 @@ module.exports = {
    */
   parse (file) {
     if (typeof file.data === "string") {
-      return file.data;
+      return {
+        data: file.data,
+        errors: null,
+      };
     }
     else if (Buffer.isBuffer(file.data)) {
-      return file.data.toString(this.encoding);
+      return {
+        data: file.data.toString(this.encoding),
+        errors: null,
+      };
     }
     else {
       throw new Error("data is not text");
```

Comment:
Should this `throw` or return null data with an item in `errors`? Basically how are throws handled versus `errors`?

Reply:
Yeah, you are right.
We shouldn't throw here.